### ISSUE TITLE
It take `kubectl` as `ash` script

### ISSUE THREAD

**Issue by wenmin-wu** [tbodt/ish#527]
I installed `kubectl` with:
```
KUBE_LATEST_VERSION="v1.16.2"
wget https://storage.googleapis.com/kubernetes-release/release/${KUBE_LATEST_VERSION}/bin/linux/amd64/kubectl -O /usr/local/bin/kubectl
```
When I run `kubectl`, it come up with:
```
teki-iPad:~# /usr/local/bin/kubectl 
@@@@@@@dd@@y3y33ss1x1x: Illegal byte sequenceE@@8@
/usr/local/bin/kubectl: line 1: I: not found
/usr/local/bin/kubectl: line 1: ELF: not found
/usr/local/bin/kubectl: line 3: syntax error: unexpected word (expecting ")")
```
seems it took `kubectl` with `ash`

**Comment by tbodt** [comment]
It's 64 bit and iSH only supports 32 bit binaries.

**Comment by wenmin-wu** [comment]
@tbodt Is there any way to support 64 bit binaries with iSH?

**Comment by tbodt** [comment]
No

**Comment by wenmin-wu** [comment]
@tbodt ok, I see, thank you very much.

**Comment by wenmin-wu** [comment]
@tbodt I downloaded a 32 bit kubectl, and come across following error:
```
wenminteki-iPad:~/k8s-client/kubernetes/client/bin# ./kubectl 
This program can only be run on processors with MMX support.
```
Will it be solved when #57 is supported? If you need any contribution or testing, pls do let me know. It's my honor to contribute to this awesome project.

**Comment by tbodt** [comment]
Indeed it would be.

**Comment by AaronForce1** [comment]
> @tbodt I downloaded a 32 bit kubectl, and come across following error:
> 
> ```
> 
> wenminteki-iPad:~/k8s-client/kubernetes/client/bin# ./kubectl 
> 
> This program can only be run on processors with MMX support.
> 
> ```
> 
> Will it be solved when #57 is supported? If you need any contribution or testing, pls do let me know. It's my honor to contribute to this awesome project.

@wenmin-wu Where did you download 32-bit kubectl?

**Comment by Teviwe** [comment]
@AaronForce1 greetings.
If your question is still relevant for you.

Follow official instructions for the 64bit kubectl installation, but change amd64 to 386:

Change this:
curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl

To:

curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/386/kubectl

Kind regards
